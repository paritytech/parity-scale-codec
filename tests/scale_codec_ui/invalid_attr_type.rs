#[derive(::parity_scale_codec::Encode)]
#[codec(crate = ::parity_scale_codec)]
pub enum Enum {
    #[codec(index = "invalid")]
    Variant1,
}

fn main() {}
