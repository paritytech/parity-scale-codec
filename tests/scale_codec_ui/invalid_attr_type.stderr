error: Error checking variant attribute: Only u8 indices are accepted for attribute variant `#[codec(index = $u8)]`
 --> tests/scale_codec_ui/invalid_attr_type.rs:4:5
  |
4 |     #[codec(index = "invalid")]
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^
